tools/bumper/parser_test.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Testing internal parser", func() {
+
+	var tmpfile *os.File
+
+	CreateTempFileWithContent := func (content []byte) {
+		var err error
+		tmpfile, err = ioutil.TempFile("", "test")
+		Expect(err).ToNot(HaveOccurred(), "Should successfully create temp file")
+
+		By("Writing and closing the temporary file")
+		_, err = tmpfile.Write(content)
+		Expect(err).ToNot(HaveOccurred(), "Should successfully write content to the temp file")
+
+		err = tmpfile.Close()
```

Comment:
This has to be done at `defer` or `AfterEach` somehow

Reply:
`Close` only closed the file descriptor, I already have a `Remove` in AfterEach. is it ok?

Reply:
Done